woocommerce/woocommerce
# [Experimental] Use variations to provide all the product filters wrapped by CollectionFilters block

### Changes proposed in this Pull Request:

To support discoverability and simplify adding filters this PR registers each of the new interactivity filters as a variation of the collection-filters block and inside the block we choose a template for the respective filter based on an attribute passed in the variant.

Closes https://github.com/woocommerce/woocommerce/issues/42160



### How to test the changes in this Pull Request:

1. In the block editor the filter blocks by themselves should not show in the block inserter
2. These new block variants should show
3. try adding each one of the new variants
4. It should add a collection filters wrapper and a corresponding heading for the block



### Changelog entry




<details>

#### Significance




#### Type




#### Message 
[Experimental] Use variations of collection filter to provide new interactivity based filter blocks.

#### Comment 

</details>
